### PR TITLE
updated Readme.md Installation step

### PR DESCRIPTION
**Pull request by JasonDsouza212**
This PR fixes the following issues:
- The missing step during installation has been added. 
-  Unnecessary spaces have been removed

## Changes made
- The installation process missed the step of changing the directory after cloning the repository to execute 'npm i'
- This pull request resolves the issue by removing unnecessary spaces and ensuring there are no spaces after the backtick.

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x]  All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Before
Screenshot from 2023-02-20 13-17-48

## After
Screenshot from 2023-02-20 13-17-29

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,10 +84,11 @@ Before starting out, you'll need to install the following on your computer.
 ### 🛠️ Installation
 
 1. Fork the repo
-2. Clone the repository `git clone https://github.com/WeMakeDevs/wemakedevs.git `
-3. Run `npm i` Or `npm install`to install all dependencies
-4. Run `npm start` to start the application
-5. Visit `https://localhost:3000` to view the application
+2. Clone the repository `git clone https://github.com/WeMakeDevs/wemakedevs.git`
+3. Run `cd wemakedevs` to change the working directory
+4. Run `npm i` Or `npm install` to install all dependencies
```
Comment (Chandraprakash-Darji) [inline README.md:89]: make `npm ci` and `npm clean-install` that will not generate the new package-lock file

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -84,10 +84,11 @@ Before starting out, you'll need to install the following on your computer.
...
+5. Run `npm start` to start the application
+6. Visit `https://localhost:3000` to view the application
```
[the remark this answers is not in the document]
Comment (AnuragThePathak) [reply README.md:91]: Please note that it should be http not https

**Comment by JasonDsouza212** [comment]
I think everything looks good now

File: README.md [hunk 3]
```diff
@@ -84,10 +84,11 @@ Before starting out, you'll need to install the following on your computer.
...
 1. Fork the repo
-2. Clone the repository `git clone https://github.com/WeMakeDevs/wemakedevs.git `
-3. Run `npm i` Or `npm install`to install all dependencies
-4. Run `npm start` to start the application
-5. Visit `https://localhost:3000` to view the application
+2. Clone the repository `git clone https://github.com/WeMakeDevs/wemakedevs.git`
+3. Run `cd wemakedevs` to change the working directory
+4. Run `npm ci` Or `npm clean-install` to install all dependencies
+5. Run `npm start` to start the application
+6. Visit [http://localhost:3000](http://localhost:3000) to view the application
```
Comment (Chandraprakash-Darji) [inline README.md:91]: make it https

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

**Review by Chandraprakash-Darji — APPROVED** [review]

**Pull request merged**